src/app/components/core/scheduler/update-schedule/update-schedule.component.ts

```diff
@@ -144,6 +145,9 @@ export class UpdateScheduleComponent implements OnInit, OnChanges {
             time: time.time,
             enabled: data['enabled']
           });
+          if (this.form.get('time').value === '00:00:00') {
+            this.form.controls['time'].setErrors({ invalid: true });
```

Comment:
why is this  invalid, the midnight time?

Reply:
Yes, it is invalid. Getting error from backend if time is 0.